### DISCUSSION TITLE
Notarization and 1.6.5

### DISCUSSION THREAD

**Discussion by mattahorton** [jacktrip/jacktrip#784] [General]
Hey folks,

We created the 1.6.5 release, but the notarization changes that @psiborg112 made (thank you, Aaron) to update our scripts to notarytool were failing. I have deleted that release and tag in the meantime and am working on a fix. 

Once that fix is ready, I'll put up a PR, and when it reaches main, we can recreate that release.

**Comment by psiborg112** [comment]
How's it failing? Is there a log output somewhere?

**Reply by psiborg112** [reply]
(You could always revert back to the previous script for now, but that will stop working next year, so probably better to troubleshoot sooner rather than later.)

**Reply by psiborg112** [reply]
I can't know for certain without seeing the output, but having looked at the jacktriplabs.yml file it seems that the login-keychain is possibly unset. Which is where notarytool wants to store and read credentials from by default. If you add `security login-keychain -s $KEYCHAIN_PATH` where you're setting up the keychain that should hopefully fix it. (And this would explain why the script works without issue on my system, but not on the build system.)

**Comment by ntonnaett** [comment]
If you delete a release things like #785 happen. We could have released 1.6.5.1 for fixing a macOS only bug. We should elect a maintainer and write a rule book.

**Reply by psiborg112** [reply]
Agreed. Also, there is a manual signing script which can be used to generate a signed mac package from a release in instances where the automated system goes awry. (I've shared it privately before, but have since uploaded it to github in the macos folder.)

**Reply by mattahorton** [reply]
The issue there is that none of the signed builds run if one of them fails. My bad on deleting it for sure.

**Comment by mattahorton** [comment]
Given the conversation in #785, do we need to skip version 1.6.5?

Notarization has been fixed, so I think we are ready to do whatever is next. A short list of things that I think should be done:

1. Release the existing features that were included in the original 1.6.5 attempt.
2. Merge in @psiborg112's Windows networking PR #755 
3. Do an RC pre-release with the new networking code

Version numbers there are all flexible depending on the answer to my first question.

**Reply by ntonnaett** [reply]
1.6.5 already trickled down to Fedora: https://koji.fedoraproject.org/koji/taskinfo?taskID=93537228
IMHO let's be safe and not release a second 1.6.5.

1. Release 1.6.6
2. Merge #755 
3. Prerelease 1.6.7-rc1

**Reply by ntonnaett** [reply]
If we bundle OpenSSL, let's also update to the latest version that fixes those vulnerabilities everybody is excited about.

**Reply by mattahorton** [reply]
Got it. If we feel that's a good plan, I'll open the version bump PR.

**Reply by ntonnaett** [reply]
We need to update the changelog file!

**Reply by mattahorton** [reply]
haha yes. I usually include those in the same pr.

**Reply by mattahorton** [reply]
How do we want to handle the 1.6.5 issue in that changelog?
